CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.2.1 (2019-05-29)
+
+- Fixed the text expand component to have correct spacing before the "See more" and "See less" buttons. [#38](https://github.com/blackbaud/skyux-layout/pull/38)
+- Fixed the text expand repeater component to not focus whitespace around the "See more" and "See less" buttons. [#38](https://github.com/blackbaud/skyux-layout/pull/38)
```

Comment:
By "to not focus whitespace" do we mean "to not place focus on the whitespace"?

Reply:
Yes. The focus stylings were including the whitespace to the left of the buton. I like what you have suggested.

Reply:
I'll make that change. Thanks.